exist-core/src/main/java/org/exist/security/internal/AccountImpl.java

```diff
@@ -172,8 +172,7 @@ private void instantiate(final Account from_user) throws PermissionDeniedExcepti
         //copy umask
         setUserMask(from_user.getUserMask());
 
-        if(from_user instanceof AccountImpl) {
-            final AccountImpl user = (AccountImpl) from_user;
+        if(from_user instanceof AccountImpl user) {
```

Comment:
use typeswitch here as well?

Reply:
I this I should  do that but I have no idea how to do that;
I always endup in a big sh*t where I typically have to throw away all my work and do a fresh clone.